### ISSUE TITLE
Matter Overdrive Reactor infinitly Energy

### ISSUE THREAD

**Issue by Proofstar** [Darkosto/SkyFactory4Issues#852]
Bug:
In a trial setup, I found that the Reactor already generates 200 MRf / tick with a small anomaly (941000 Matter) and only one Cobblestone per port (Fusion Reactor IO) (normal would be 4794 Rf / tick in size). The Matter generated by the Cobblestone is not consumed and the energy storage is always at 100%. So I was able to generate 2.1 trillion Rf / tick via Flux Network.

Reproduction:
Just put on a small reactor and feed it with a cobblestone. As soon as he has consumed it, he already generates the maximum amount of energy.

Game Version 4.0.4  Single/Multi Player

![2019-05-05_23 36 37](https://user-images.githubusercontent.com/49999197/57200698-2e577200-6f8f-11e9-90f1-d203efb2d30f.png)
![2019-05-05_23 36 46](https://user-images.githubusercontent.com/49999197/57200702-31526280-6f8f-11e9-9b88-8605ba590af4.png)

**Comment by LeSwede** [comment]
Yeah, my Anomaly has a Mass of 6506 and generates around 200 Million RF/t. I think the mod is behaving funny. I heard its a port from 1.7 so that could be why.

**Comment by Mstori** [comment]
Any updates on this? Still creating almost infinite energy for very cheap. Basically a creative energy cell you can craft

**Comment by SamDeAngelis** [comment]
Actually there is another problem at play here! I found this out myself with some testing. You don't even need ANY of that stuff, all you need is the reactor block itself. You see, you can "Charge" the reactor block with energy that you already have and the reactor will output that exact amount of energy back out. You can use this on a reactor block to charge itself up to 200mill rf/t! You don't need any other blocks and can charge other reactors with each one. I made a line of about 50 reactor blocks and charged a mekanism battery at over 10 Billion rf/t!

In this example I used a reactor block and a single solar panel to charge a Huge battery instantly!
![2020-09-01_22 50 36](https://user-images.githubusercontent.com/43426023/91926860-353ae580-eca6-11ea-826a-59ea5068adca.png)
![2020-09-01_22 50 49](https://user-images.githubusercontent.com/43426023/91926864-39ff9980-eca6-11ea-82f4-c8c054bc791e.png)
![2020-09-01_22 50 56](https://user-images.githubusercontent.com/43426023/91926873-3e2bb700-eca6-11ea-8982-360fc5f4fdcd.png)

This took about 10 seconds re create!

**Comment by infiniteblock** [comment]
This seems to be fixed in the community edition.
